sites/public/lib/HorizontalScrollSection.tsx

```diff
@@ -60,6 +62,7 @@ const HorizontalScrollSection = (props: HorizontalScrollSectionProps) => {
           className={styles.title__button}
           onClick={leftButtonClick}
           disabled={!canScrollLeft}
+          ariaLabel="Scroll left"
```

Comment:
Should these strings be translated?

Reply:
I filed https://github.com/CityOfDetroit/bloom/issues/755 . I'm putting it off for now since we want to move the whole HorizontalScrollSection to ui-components so I'll add it when we move it there since they have a different set of translations